documentation/getting_started_with_bolt.md

```diff
@@ -46,11 +46,11 @@ Use the `bolt project init` command to create a project directory named
 bolt project init ./my_project
 ```
 
-Listing the contents of `my_project` shows a `bolt.yaml` file:
+Listing the contents of `my_project` shows a `bolt-project.yaml` file:
```

Comment:
`bolt project init` currently creates a bolt.yaml file, so these instructions need to change. Maybe we should hold off on edits to the GS until that command is in place? 

Alternatively, we can change these instructions to direct the user to create the directory and bolt-project.yaml manually, and enter a name for their project in the file. And then I'll work on the update ticket in #2017 ASAP

Reply:
Thought about this some more. Let's replace these instructions for now. Let's replace lines 42-48 with:

1. Create a directory named `my_project`:
   ```bash
   mkdir my_project
   ```
2. Create a file named `bolt-project.yaml` with the following contents:
   ```yaml
   # my_project/bolt-project.yaml
   name: my_project
   ```